README.md

```diff
@@ -0,0 +1,78 @@
+# kmesh-coredns-plugin
+
+The plugin runs as a standlone server in the cluster, serving DNS A records over gRPC to CoreDNS.
+
+# Quick start
+
+## Step 1: Install kubectl & kind
+
+Linux:
+```sh
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.18.0/kind-linux-amd64
+chmod +x ./kubectl ./kind
+sudo mv ./kubectl ./kind /usr/local/bin
+```
+
+## Step 2: Create and Activate kind
+```sh
+kind create cluster --name kmesh-dns
+```
+
+## Step3: Install Istio
```

Comment:
seems we donot need to specify the install step 1 ~ step 3

Reply:
yes, I will remove that if we assume that kmesh has installed